.github/workflows/qodana.yml

```diff
@@ -6,22 +6,20 @@ on:
   push:
     branches:
       - master
+
 jobs:
-  inspection:
+  qodana:
```

Comment:
I think this can be another name
![image](https://user-images.githubusercontent.com/9636937/203816585-870dfdba-3d92-4df4-a857-cad24fc18f6c.png)